neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -172,12 +172,17 @@ def home(self, request):
             return self.get_data(content)
 
         elif "OPTIONS" == request.method.decode("utf-8"):
-            return {'supported HTTP methods': ("GET", "POST"),
-                    'JSON-RPC server type': "default"}
+            return self.options_response()
 
         error = JsonRpcError.invalidRequest("%s is not a supported HTTP method" % request.method.decode("utf-8"))
         return self.get_custom_error_payload(request_id, error.code, error.message)
 
+    @classmethod
+    def options_response(self):
```

Comment:
The argument should be `cls` rather than `self`

Reply:
Nice! Thanks

Reply:
Same in `neo/api/JSONRPC/ExtendedJsonRpcApi.py` 😅

Reply:
I wondered about that but does that mean test_json_rpc_api.py is wrong here https://github.com/CityOfZion/neo-python/blob/e3780126f9cf8175f9f3c7ad27f36f2baae20508/neo/api/JSONRPC/test_json_rpc_api.py#L37?

Reply:
Technically the parameter can be named whatever you want, but it's a convention to name it `cls` for class methods as it is a convention to name the first parameter `self` for instance methods.
https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments